public/templates/demo.html

```diff
@@ -92,38 +108,34 @@
       </div>
     </header>
     <!-- ***** Header Area End ***** -->
-
+     <!-- ***** Context Text Start ***** -->
+      <div class = "context">
+        <h1>View Our <strong>Demo</strong>!</h1>
+        <p>
+          This is a demo of a conversation between a patient and a doctor.
+          Use this only as a basic reference for our site's capabilities.
+          Once you're matched with a doctor, you will recieve true medical 
+          advice and engage in a more helpful conversation! 
+        </p>
+      </div>
+    <!-- ***** Context Text End ***** -->
+    <!-- ***** Demo Videos Start ***** -->    
     <div class="images_container">
-      <video autoplay class="video">
+    <!-- video1 is the video on the left side
+      video2 is the video on the right side -->
+      <video autoplay class="video" id="video1">
         <source
-          src="https://cdn.glitch.com/eb6e6852-d15d-4972-9a23-18fd43f55592%2Fvideo1.mov?v=1598213723677"
+          src="/assets/videos/demoVideo1.mov"
```

Comment:
Why not give the videos more descriptive names? Example instead of `demoVideo1` how about `patientVideo` ditto for the practitioner video. That way the video and element IDs are self describing.

Reply:
Were you able to make the changes requested here?